langchain/package.json

```diff
@@ -1149,6 +1155,7 @@
     "openapi-types": "^12.1.3",
     "p-queue": "^6.6.2",
     "p-retry": "4",
+    "portkey-ai": "^0.1.11",
```

Comment:
Can you make this a dev dependency + peer dependency and mark it as optional under `peerDependenciesMeta`?

Reply:
See guide here: https://github.com/langchain-ai/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md